syntax = "proto3";

message DeviceState {
	string DeviceIdentity = 1;   
	string LastEvidence = 2;
	string State = 3;
	int32 Request = 4;
}
